deploy/kubernetes/rbac-external-provisioner.yaml

```diff
@@ -59,7 +59,7 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 
 ---
-# Provisioner must be able to work with endpoints in current namespace
+# Provisioner must be able to work with endpoints and leases in current namespace
```

Comment:
@xing-yang why do we need external-provisioner examples here?  shouldn't we be referring to external-provisioner repo?

Reply:
This is just easier for folks who want to know how to get started with snapshot. They look at this one repo and can find an example of how to test snapshot, without going through multiple repos.

Reply:
The difficulty is keeping it up to date as external-provisioner gets updated.

Reply:
If we don't update the provisioner image, these rbac files will stay the same.  If we need to update the image, we'll need to update the rbac files. We have a note in README to remind folks to refresh the rbac files when the provisioner image is updated: https://github.com/kubernetes-csi/external-snapshotter/tree/master/deploy/kubernetes